post_installation_configuration/configuring-multi-arch-compute-machines/multi-architecture-configuration.adoc

```diff
@@ -22,21 +22,75 @@ For information on migrating your single-architecture cluster to a cluster that
 
 == Configuring your cluster with multi-architecture compute machines
 
-To create a cluster with multi-architecture compute machines for various platforms, you can use the documentation in the following sections:
+To create a cluster with multi-architecture compute machines with different installation options and platforms, you can use the documentation in the following table:
 
-* xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-azure.adoc#creating-multi-arch-compute-nodes-azure[Creating a cluster with multi-architecture compute machines on Azure]
+.Cluster with multi-architecture compute machine installation options
+[cols="3,1,1,1,1,1",options="header"]
+|===
+|Documentation section |Platform |User-provisioned installation |Installer-provisioned installation |Control Plane |Compute node
 
-* xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-aws.adoc#creating-multi-arch-compute-nodes-aws[Creating a cluster with multi-architecture compute machines on AWS]
+|xref:../../post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-azure.adoc#creating-multi-arch-compute-nodes-azure[Creating a cluster with multi-architecture compute machines on Azure]
+|Microsoft Azure
+|
+|&#10003;
+|`x86_64`
+|`ARM64`
```

Comment:
@kelbrown20 I think we should either use `amd64` + `arm64` OR `x86_64` + `aarch64`? We don't want to mix the use of rpm arch with goarch?